packs/feats/fresh-ingredients.json

```diff
@@ -31,7 +31,7 @@
         "rules": [
             {
                 "key": "FlatModifier",
-                "label": "Spend a batch of herbs in wilderness",
+                "label": "Fresh ingredients in the wilderness",
```

Comment:
We'll probably want to localize the label.

Reply:
Would you like me to do that within this PR? If so, could you give me an example?

Reply:
For a label Syntax option example, check out the Roll Option on Clandestine Cloak (either version).

To add the localization, go to your local repo and find the `re-en.json` file. (Located under pf2e -> static ->lang) and the the entry. Something like this:

```json
"Herbalist" {
    "FreshIngredients": {
         "FlatModifierLabel": "Fresh ingredients"
     },
      "FreshIngredientsWild": {
            "FlatModifierLabel": "Fresh ingredients in the wilderness"
     }
}
```

 Once you add it to the file, just save the file on the branch the changes should populate. On the feat, at the path to the localization (again, refer to Clandestine Cloak's roll option on those are set).